src/app.module.ts

```diff
@@ -131,15 +124,7 @@ import { MaskSensitiveDataInterceptorModule } from "./common/interceptors/mask-s
           },
           template: {
             dir: join(__dirname, "./common/email-templates"),
-            adapter: new HandlebarsAdapter({
-              unwrapJSON: unwrapJSON,
-              keyToWord: formatCamelCase,
-              eq: (a, b) => a === b,
-              jsonify: jsonify,
-              job_v3: job_v3,
-              urlencode: urlencode,
-              base64enc: base64enc,
-            }),
+            adapter: new HandlebarsAdapter(handlebarsHelpers),
```

Comment:
for a later PR maybe, should we move the content of `useFactory` to a dedicated mailer factory class/file?